base/docs/helpdb/Base.jl

```diff
@@ -2603,7 +2603,8 @@ creates a task, and does not run it.
 """
     fld(x, y)
 
-Largest integer less than or equal to `x/y`.
+Largest integer `n` such that `y * n` is less than or equal to `x`. Note that this can be
```

Comment:
The generic definition is
```Julia
fld{T<:Real}(x::T,y::T) = convert(T,round((x-mod(x,y))/y))
```
where `mod` could presumably be implemented as
```
mod{T<:Real}(x::T,y::T) = x-y*floor(x/y)
```
If you remove some extraneous gymnastics (multiply by / divide by `y`, subtract from `x` twice) and ignore the associated rounding errors, this leads to
```Julia
fld(x,y) = floor(x/y)
```
which is probably also what most people expect as it is direct and simple. "fld" means "flooring division", so this is the definition I'd suggest to use. (And then we can update and simplify the code, and add tests for cases where round-off errors kick in.)